### PR TITLE
generate: add specified cgroups path support

### PR DESCRIPTION
**Pull request by Mashimiao**
Signed-off-by: Ma Shimiao <mashimiao.fnst@cn.fujitsu.com>

### CODE REVIEW

File: man/ocitools-generate.1.md [hunk 1]
```diff
@@ -47,6 +47,10 @@ compatible runtime like runC to run a container.
 **--cwd**=PATH
   Current working directory for the process
 
+**--cgroups-path**=""
```
Comment (wking) [inline man/ocitools-generate.1.md]: “cg” comes before “cw” in alphabetical order, so this entry should land between `--cap-drop` and `--cwd`.

File: man/ocitools-generate.1.md [hunk 2]
```diff
@@ -47,6 +47,10 @@ compatible runtime like runC to run a container.
...
+  Specifiy the path to the cgroups and it is expected to be relative 
+  to the cgroups mount point.
```
Comment (wking) [inline man/ocitools-generate.1.md]: I'd reword to:

> Specifiy the path to the cgroups relative to the cgroups mount point.

**Comment by wking** [comment]
I left two minor nits, but otherwise f5b89dd2 looks good to me.

**Comment by Mashimiao** [comment]
@wking  Thanks for review. Fixed.

**Comment by liangchenye** [comment]
afa3fce LGTM

**Comment by mrunalp** [comment]
LGTM
